client/controllers/events/eventAffectedAreas.coffee

```diff
@@ -153,7 +153,8 @@ Template.eventAffectedAreas.onRendered ->
     EventIncidents.find(@data.filterQuery.get()).map (incident) ->
       incident.locations.forEach (location) ->
         key = "#{location.latitude},#{location.longitude}"
-        incidentsByLatLng[key] = (incidentsByLatLng[key] or []).concat(incident)
+        if not incidentsByLatLng[key]
```

Comment:
I'm not seeing the duplicate id warning. Could you give an example of how to reproduce it?